packages/web-components/src/components/ic-search-bar/ic-search-bar.tsx

```diff
@@ -509,23 +512,41 @@ export class SearchBar {
       ".search-results-status"
     ) as HTMLParagraphElement;
 
-    if (!this.open || this.value === "") {
+    if (
+      !this.open ||
+      this.value === "" ||
+      this.value.length < this.charactersUntilSuggestion
+    ) {
       searchResultsStatusEl.innerText = "";
     } else if (
-      this.options.length > 0 &&
+      this.hasOptionsOrFilterDisabled() &&
       this.filteredOptions.length > 0 &&
       this.open &&
       searchResultsStatusEl
     ) {
-      searchResultsStatusEl.innerText = `${this.filteredOptions.length} result${
-        this.filteredOptions.length > 1 ? "s" : ""
-      } available`;
+      if (this.hadNoOptions()) {
+        searchResultsStatusEl.innerText = this.emptyOptionListText;
+      } else {
+        searchResultsStatusEl.innerText = `${
+          this.filteredOptions.length
+        } result${this.filteredOptions.length > 1 ? "s" : ""} available`;
+      }
     }
   };
 
   private hasOptionsOrFilterDisabled = (): boolean =>
     this.options.length > 0 || this.disableFilter;
 
+  private hadNoOptions = (): boolean =>
+    this.filteredOptions.length > 0 &&
```

Comment:
Should we make this stricter and see if it's equal to 1 as there should only be one item when 'no results' is displayed